bundles/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/internal/SatelBinding.java

```diff
@@ -0,0 +1,235 @@
+/**
+ * Copyright (c) 2010-2014, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.satel.internal;
+
+import java.util.ArrayList;
+import java.util.Dictionary;
+import java.util.List;
+
+import org.apache.commons.lang.StringUtils;
+import org.openhab.binding.satel.SatelBindingConfig;
+import org.openhab.binding.satel.SatelBindingProvider;
+import org.openhab.binding.satel.internal.event.EventListener;
+import org.openhab.binding.satel.internal.event.NewStatesEvent;
+import org.openhab.binding.satel.internal.event.SatelEvent;
+import org.openhab.binding.satel.internal.protocol.Ethm1Module;
+import org.openhab.binding.satel.internal.protocol.IntRSModule;
+import org.openhab.binding.satel.internal.protocol.SatelMessage;
+import org.openhab.binding.satel.internal.protocol.SatelModule;
+import org.openhab.binding.satel.internal.protocol.command.NewStatesCommand;
+import org.openhab.binding.satel.internal.types.IntegraType;
+import org.openhab.core.binding.AbstractActiveBinding;
+import org.openhab.core.items.Item;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.osgi.service.cm.ConfigurationException;
+import org.osgi.service.cm.ManagedService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This is main service class that helps exchanging data between OpenHAB and
+ * Satel module in both directions. Implements regular OpenHAB binding service.
+ * 
+ * @author Krzysztof Goworek
+ * @since 1.7.0
+ */
+public class SatelBinding extends AbstractActiveBinding<SatelBindingProvider> implements ManagedService, EventListener {
+
+	private static final Logger logger = LoggerFactory.getLogger(SatelBinding.class);
+
+	private long refreshInterval = 10000;
+	private String userCode;
+	private SatelModule satelModule = null;
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	protected String getName() {
+		return "Satel Refresh Service";
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	protected long getRefreshInterval() {
+		return refreshInterval;
+	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	@Override
+	public void execute() {
+		if (!isProperlyConfigured()) {
+			logger.warn("Binding not properly configured, exiting");
+			return;
+		}
```

Comment:
this can be removed since the polling mechanism is shutdown once setProperlyConfigured(false) has been called (see AbstractActiveBinding.setProperlyConfigured())